source/UniqueItemStuff.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BattleTech;
+using BattleTech.UI;
+
+namespace MechEngineer
+{
+    public partial class MechEngineerSettings
+    {
+        public UniqueItem[] Uniques =
+        {
+            //new UniqueItem {ItemPrefix = "emod_armor_", ReplaceTag = "Armor"},
+            //new UniqueItem {ItemPrefix = "emod_structure_", ReplaceTag = "Structure"},
+            new UniqueItem {ItemPrefix = "emod_engine_", ReplaceTag = "Engine Core"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_std_center", ReplaceTag = "Engine Shielding"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_center", ReplaceTag = "Engine Shielding"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_left", ReplaceTag = "Engine Shielding Left"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_right", ReplaceTag = "Engine Shielding Right"},
+            new UniqueItem {ItemPrefix = "emod_engineslots_xl_right", ReplaceTag = "Engine Shielding Right"},
+            new UniqueItem {ItemPrefix = "Gear_Gyro_", ReplaceTag = "Gyro"},
+            new UniqueItem {ItemPrefix = "Gear_Cockpit_", ReplaceTag = "Cockpit"},
+        };
+
+        public UniqueCategory[] UniqueCategories =
+        {
+            //new UniqueCategory {Tag = "Armor"},
+            //new UniqueCategory {Tag = "Structure"},
+            new UniqueCategory {Tag = "Engine Core"},
+            new UniqueCategory {Tag = "Engine Shielding"},
+            new UniqueCategory {Tag = "Gyro"},
+            new UniqueCategory {Tag = "Cockpit"},
+            new UniqueCategory {Tag = "Engine Shielding Left", Required = false},
+            new UniqueCategory {Tag = "Engine Shielding Right", Required = false},
+        };
+    }
+
+
+    public class UniqueCategory
+    {
+        /// <summary>
+        /// catefory Name
+        /// </summary>
+        public string Tag;
+        /// <summary>
+        /// for validation checks - need this installed
+        /// TODO: not implemented!
+        /// </summary>
+        public bool Required = true;
```

Comment:
if this would be the final change, I would say remove this, but I know you want to complete the implementation eventually. so we can keep it in the PR

Reply:
implemented